language/stdlib/modules/references_template.md

```diff
@@ -1,2 +1,2 @@
-
-[]: # (File containing markdown style reference definitions to be included in each generated doc)
```

Comment:
This is actually a comment. It looks funny, but this is the recommended way to make comments in Markdown. Please leave it.

Reply:
Added it back.

Reply:
Oops! Just seeing now that this is printed in the documentation, unexpectedly. Yes please remove it, I care later for possibly bringing it back. I hope it doesn't appear in the way you have added it.

Reply:
Ah, OK. I removed it.